### ISSUE TITLE
Banning devices Such as Pixy and ect.

### ISSUE THREAD

**Issue by ZvonoBednarcik** [RoboCupJuniorTC/soccer-rules#30]
### Please describe your suggestion in one or two sentences

I think devices used for detecting the ball in Soccer Open league should be banned. By this I mean that all programs that are used for camera detection should be programmed exclusively by the students. This doesn't include cameras used in lightweight for locating the goal of the opponent.

### Please explain why do you think it should be in the rules

One of the main reasons why the passive ball in RCJ Soccer Open was introduced was to close the gap between RCJ and The regular RC. Instead of picking up new skills (programming camera detection), many of the teams use devices such as pixy, and they learn nothing new. I think that all of the teams should be able to explain the algorithm used into detail. For example saying "Device "X" send the position of the ball to micro controller "X"" would not be enough. A more complex explanation would be required: "Device X takes frame from camera in the form of a 3 dimensional RGB array converts to HSV filter out colors into binary image, then finds topmost bottom-most leftmost and rightmost pixel, averages them out and than uses the I2C protocol to send to micro-controller "X" ". This way all participants of Soccer Open would legitimately have to learn something new, which is always good.

**Comment by mrshu** [comment]
Hi @ZvonoBednarcik,

First of all, I am sorry for my late reply.

**tl;dr**: *It would be dangerous to change the rules abruptly, but a change like this may come eventually.*

In general terms, I do agree with this sentiment (i.e. that devices such as Pixy do not force participants to learn something fundamentally new). However, we also need to consider that the Open league is still quite young and that if the requirements "jumped" too much from one year to the other, many teams may not be able to follow, which would be a net loss for the whole competition, at least in my opinion.

That being said, however, I do believe a rule like this will be at least considered for the next years. Note that as many advanced participants will be "forced" to move to the Open league (by the participation limit rule),  the Open league will be able to "up its game" much more. One of the small steps into this direction are the Technical Challenges, which should be released soon.

Thanks again for creating an issue and please feel free to create another one in the future!
